packages/juggler/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@loopback/juggler",
```

Comment:
It makes me wonder whether `juggler` is the best name for this package. Have we considered renaming it, e.g. to `@loopback/persistence`, `@loopback/orm`, `@loopback/data-access`, `@loopback/model` or something else?